src/js/lineChartTimeSeriesMultiDataSet.js

```diff
@@ -0,0 +1,184 @@
+            interpolation: "linear",
+            // In milliseconds
+            transitionLength: 2000
+        },
+        scaleOptions: {
+            // transform rules to apply to yScale min
+            yScaleMinTransform: {
+                "literalValue": 0
+            },
+            // transform rules to apply to yScale max
+            yScaleMaxTransform: {
+                "transform": {
+                    "type": "fluid.transforms.binaryOp",
+                    "leftPath": "max",
+                    "right": 1.25,
+                    operator: "*"
+                }
+            }
+        },
+        events: {
+            onChartCreated: null,  // Fire when the line is created. Ready to register D3 DOM event listeners,
+            onDraw: null
+        },
+        listeners: {
+            "onCreate.create": {
+                funcName: "floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.create",
+                args: ["{that}"]
+            }
+        },
+        modelListeners: {
+            dataSet: [{
+                funcName: "{that}.events.onDraw.fire",
+                excludeSource: "init"
+            }]
+        },
+        invokers: {
+            getXScale: {
+                funcName: "floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.getXScaleTimeSeries",
+                args: ["{that}"]
+            },
+            getYScale: {
+                funcName: "floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.getYScale",
+                args: ["{that}"]
+            }
+        }
+    });
+
+    floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.fixedValue = function (fixedValue) {
+        return fixedValue;
+    };
+
+    floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.multiplyValue = function (value, multiplier) {
+        return value * multiplier;
+    };
+
+    // Accumulator function for consolidating multiple dataset items together
+    // for purposes of determining max/min out a group of datasets
+    floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.concatData = function (setItem, accumulationArray) {
+        return accumulationArray.concat(setItem.data);
+    };
+
+    floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.getYScale = function (that) {
+        var height = that.options.svgOptions.height;
+        var padding = that.options.lineOptions.padding;
+        var dataSet = that.model.dataSet;
+
+        // Create an array consisting of all the values in every dataset array
+        var combinedData = fluid.accumulate(dataSet, floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.concatData, []);
+
+        // Get the min value of that combined array
+        var minValue = d3.min(combinedData, function (d) {
+            return d.value;
+        });
+
+        // Get the max value of that combined array
+        var maxValue = d3.max(combinedData, function (d) {
+            return d.value;
+        });
+
+        var rawYScale = {
+                min: minValue,
```

Comment:
I'm surprised the linting doesn't capture a indentation issue like this, as well as line 129-131.